democracy/views/section_comment.py

```diff
@@ -251,9 +251,11 @@ def get_can_delete(self, comment: SectionComment):
     def to_representation(self, instance):
```

Comment:
No need to show these fields to anyone (according to smoi) unless FE needs them. Checked that FE does not need them. So could could just use `settings.HEARING_REPORT_PUBLIC_AUTHOR_NAMES` to determine whether to include `creator_email` in the serializer or not.